theano/tensor/nnet/__init__.py

```diff
@@ -30,3 +30,91 @@
+        You can give ``None`` for any element of the list to specify that this
+        element is not known at compile time.
+
+    :type filter_shape: None, tuple/list of len 4 of int or Constant variable
+    :param filter_shape: The shape of the filters parameter.
+        Optional, possibly used to choose an optimal implementation.
+        You can give ``None`` for any element of the list to specify that this
+        element is not known at compile time.
+
+    :type border_mode: str, int or tuple of two int
+    :param border_mode: Either of the following:
+        * ``'valid'``: apply filter wherever it completely overlaps with the
+          input. Generates output of shape: input shape - filter shape + 1
+        * ``'full'``: apply filter wherever it partly overlaps with the input.
+          Generates output of shape: input shape + filter shape - 1
+        * ``'half'``: pad input with a symmetric border of ``filter rows // 2``
+          rows and ``filter columns // 2`` columns, then perform a valid
+          convolution. For filters with an odd number of rows and columns, this
+          leads to the output shape being equal to the input shape.
+        * ``int``: pad input with a symmetric border of zeros of the given
+          width, then perform a valid convolution.
+        * ``(int1, int2)``: pad input with a symmetric border of ``int1`` rows
+          and ``int2`` columns, then perform a valid convolution.
+
+    :type subsample: tuple of len 2
+    :param subsample: factor by which to subsample the output.
+        Also called strides elsewhere.
+
+    :type filter_flip: bool
+    :param filter_flip: If ``True``, will flip the filter rows and columns
+        before sliding them over the input. This operation is normally referred
+        to as a convolution, and this is the default. If ``False``, the filters
+        are not flipped and the operation is referred to as a cross-correlation.
+
+    :rtype: symbolic 4D tensor
+    :return: set of feature maps generated by convolutional layer. Tensor is
+        of shape (batch size, output channels, output rows, output columns)
+    """
+
+    if len(kargs.keys()) > 0:
+        warnings.warn(str(kargs.keys()) +
+                      " are now deprecated in "
+                      "`tensor.nnet.abstract_conv.conv2d` interface"
+                      " and will be ignored.")
+
+    if image_shape is not None:
+        warnings.warn("image_shape is no longer supported in "
+                      "`tensor.nnet.abstract_conv.conv2d` interface"
+                      " use input_shape instead.")
+        if input_shape is None:
+            input_shape = image_shape
+        else:
+            raise ValueError("input_shape and image_shape should not"
+                             " be provided at the same time.")
+
+    return abstract_conv2d(input, filters, input_shape, filter_shape,
+                           border_mode, subsample=(1, 1))
```

Comment:
o.O Apparently, as we say in french, I didn't have my eyes aligned with my sockets when I reviewed this... Thanks so much for the fix.